pkg/kubelet/kubelet.go

```diff
@@ -795,6 +802,9 @@ func (kl *Kubelet) initialNodeStatus() (*api.Node, error) {
 			Name:   kl.nodeName,
 			Labels: map[string]string{"kubernetes.io/hostname": kl.hostname},
 		},
+		Spec: api.NodeSpec{
```

Comment:
@mikedanese: What is the use case for making `--register-schedulable` flag an initialization only option instead of being valid for the entire lifetime? 
In general, I find multiple ways to configure the same behavior confusing. As a user, I'd prefer to either flip a flag or post an update to an API server object and not deal with how these two options interact.